src/components/table/read-only/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { FC, useCallback, useRef, useState } from 'react';
+import { Box } from 'rebass';
+
+// Components
+import Table, { TableProps } from '../index';
+
+// Utils
+import { getColumns, getRows } from './utils';
+
+// Styles
+import {
+  containerStyles,
+  dropdownStyles,
+  tableStyles,
+  theadStyles,
+  trowStyles
+} from '../table.styles';
+
+// Hooks
+import { ListItem, useDropdown, useOnClickOutside } from '../../..';
+import List from '../../list/container';
+
+export interface TheadProps{
+  column: string;
+  className: string;
+  actions: Array<{
+    label: string,
+    handler: (column: string) => void
+  }>
+}
+
+const Thead: FC<TheadProps> = ({
+  column,
+  className,
+  actions,
+}: TheadProps) => {
+
+  const containerRef = useRef(null);
+  const [isOpen, handleToggle, handleClickOutside] = useDropdown();
+  useOnClickOutside<HTMLDivElement>(containerRef, handleClickOutside);
+
+  const handleToggleList = useCallback(() => {
```

Comment:
"handleToggleList" is redundant, "handleToggle" is already wrapped with useCannback. You can just pass "handleToggle" function directly.